.ci/docker-compose-file/.env

```diff
@@ -12,6 +12,7 @@ OPENTS_TAG=9aa7f88
 KINESIS_TAG=2.1
 HSTREAMDB_TAG=v0.19.3
 HSTREAMDB_ZK_TAG=3.8.1
+DATALAYERS_TAG=nightly
```

Comment:
I think it would be best to pin a fixed version here.  Otherwise, it might break CI and take some time to realize it's due to an upstream image update.

Reply:
Of course, I'll send another PR to update once the Datalayers have a new stable tag.